docs/navigation-ux-analysis.md

```
@@ -0,0 +1,637 @@
+| Profile page is a stub | [#5 Build "You" page](issues/issue-5-build-you-profile-page.md) | P1 |
+| No notifications page | [#6 Notifications inbox](issues/issue-6-notifications-inbox-page.md) | P1 |
+| Friends wastes a tab slot | [#7 Merge into Messages](issues/issue-7-merge-friends-into-messages-tab.md) | P1 |
+| No mobile search entry point | [#8 Mobile search](issues/issue-8-mobile-search-affordance.md) | P2 |
+
+---
+
+## 7. Recommendation
+
+### Target Bottom Nav
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)